src/components/Terminal.jsx

```diff
@@ -287,48 +295,48 @@ export default class Terminal extends React.Component {
       },
       prompt: {
         ...sourceStyles.prompt,
-        color: this.props.promptLabelColor || '#EE9C34'
```

Comment:
Where is the new default colour defined?

Reply:
Maybe should put those default string in one place, easier to read and modify.